Source/MediaInfo/MediaInfo_Config.cpp

```diff
@@ -295,6 +431,10 @@ Ztring MediaInfo_Config::Option (const String &Option, const String &Value_Raw)
         //Merge
         Value=FromFile;
     }
+    else if (Value_Raw.find(__T("cstr://"))==0)

```

Comment:
`Value_Raw` may be relatively big, doing a `find` may have performance impact with some corner cases, I am trying to avoid such impact in new code.
Please something e.g. [std::equal](http://stackoverflow.com/a/8596924/3882655).